yspryn/checkio_tasks/end_zero_hw5.py

```diff
@@ -0,0 +1,11 @@
+# Try to find out how many zeros a given number has at the end.
+
+
+def end_zeros(a: int) -> int:
+    res = 0
+    for i in reversed(str(a)):
+        if i == "0":
+            res += 1
+        else:
+            break
+    return res
```

Comment:
rstrip

Reply:
```
def end_zeros(a: int) -> int:
    res = 0
    for i in reversed(str(a)):
        if i != "0":
            break
        res += 1
    return res


```

Reply:
you can use 
enumerate(reversed(str(a)), start=1).   
it will serve as a counter